### PR TITLE
Images fixes

### PR DESCRIPTION
**Pull request by paultalarian**
- Replace div by figure
- Remove the figure wrapper
- Passed width as a parameter to the image to fix the 100% case that was breaking the layout
- Removed some unused code that make the commit fail

/!\ it impacts all kind of assets, I tested images and videos. To be double checked

### CODE REVIEW

File: packages/react-notion-x/src/components/asset.tsx [hunk 1]
```diff
@@ -269,17 +269,18 @@ export const Asset: React.FC<{
         alt={alt}
         zoomable={zoomable}
         height={style.height as number}
+        width={style.width as number}
```
Comment (odemian) [inline packages/react-notion-x/src/components/asset.tsx:272]: @paultalarian doesn't width require units (like px)?

Comment (odemian) [reply packages/react-notion-x/src/components/asset.tsx:272]: From what I see, the width and height proeprties are strings (the initial value ff width is '100%', which is string). Are we sure we are not breaking something here by forcing wrong type?

**Comment by odemian** [comment]
@paultalarian the `react-lazy-images` library was not updated for 4 years. We will better to switch to the [next/image](https://nextjs.org/docs/api-reference/next/image) in the future

File: packages/react-notion-x/src/components/asset.tsx (hunk 1 above)
Comment (paultalarian) [reply packages/react-notion-x/src/components/asset.tsx:272]: Good question, height was already defined as is, so I just followed the same logic. 
From my tests, everything works fine, but I agree it looks odd.

**Comment by paultalarian** [comment]
Good point yes, thanks.

**Pull request merged**